neo/Ledger/MemoryPool.cs

```diff
@@ -16,32 +16,33 @@ public class MemoryPool : IReadOnlyCollection<Transaction>
     {
         private class PoolItem : IComparable<PoolItem>
         {
-            public readonly Transaction Transaction;
+            public readonly Transaction Tx;
             public readonly DateTime Timestamp;
             public DateTime LastBroadcastTimestamp;
```

Comment:
Can you add a comment here explaining the meaning of Timestamp and LastBroadcastTimestamp please? I think Timestamp is the time the item was received, right? And LastBroadcastTimestamp is the last time it was sent to other nodes?

Reply:
Your thinking is correct. I can add a comment.